frontend/awx/views/jobs/JobOutput/JobOutput.cy.tsx

```diff
@@ -1,42 +1,183 @@
-import { JobOutputInner as JobOutput } from './JobOutput';
-import job from '../../../../../cypress/fixtures/job.json';
-import type { Job } from '../../../interfaces/Job';
-
-describe('JobOutput.cy.tsx', () => {
-  beforeEach(() => {
-    cy.intercept(
-      {
-        method: 'GET',
-        url: '/api/v2/jobs/26/job_events/?order_by=counter&page=1&page_size=50',
-        hostname: 'localhost',
-      },
-      {
-        fixture: 'jobEvents.json',
+import { useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
@jamestalton was this filed modified intentionally? It does not look the original set of tests.

Reply:
DOH. Nope, thanks! Fixing